### DISCUSSION TITLE
How allow an user to create a document with an id of another document?

### DISCUSSION THREAD

**Discussion by realtebo** [appwrite/appwrite#3216] [Q&A]
> Sorry, bad question title. I know.

user can create a group. Only one. And it 'joins' automatically to this group. So user->group_id is setup . ok.

the user create an 'item', Item should have the user_id and/or the group_id automatically. and must not be able to create 'items' using different user_id than its id and cannot create items using group_id it is not part of.

Is it a bit clear?

Is this possibile using app-write? 

can I use, for example, 'per document' permission when creating a new 'item'  checking the group_id used? if yes... how...

**Comment by maeddin** [comment]
No, this is not possible with the AppWrite system. You have to do it manually via AppWrite Functions.
